### PR TITLE
i9/Replace pre-commit manager

### LINKED ISSUE #9: Replace pre-commit function and document it

### PR DESCRIPTION
**Pull request by Dentsor**
Replace husky with pre-commit.com

closes #9

### CODE REVIEW

**Comment by LarsSelbekk** [comment]
No virtual environment?

**Comment by LarsSelbekk** [comment]
Should also use the same configuration for format and pre-commit.

**Comment by Dentsor** [comment]
Depends on #23

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Speiderapp
+
+Speiderapp is a mobile-first webapp for (norwegian) scouts.
```
Comment (LarsSelbekk) [inline README.md:3]: Typo: "Norwegian" (capital N)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+
+# Common setup
+
+## Requirements
+This project expect the use of a Unix-like, like Mac OS, Linux or WSL.
+It can probably run on Windows/others too, but this may need adapting some commands.
+
+## Use of Makefile
+This project is using a �makefile� for handling setup and common tasks.
```
Comment (LarsSelbekk) [inline README.md:16]: The "`"-characters seem to be in the windows 1251-encoding instead of utf8.

**Comment by Dentsor** [comment]
Naming rule violation, IDE1006

**Review by LarsSelbekk — APPROVED** [review]
Great. It doesn't seem like it catches errors funnily enough, but that's out-of-scope I guess.

**Pull request merged**